docs(root): add guidance for using SlottedSVG and update code examples (#259)
mi6/ic-design-system: src/content/structured/get-started/react.mdx

The `@ukic/react` package will need to be transformed before you can use these components in Jest tests.

Add a `transformIgnorePatterns` field with the value `["/node_modules/(?!@ukic/react)"]` to your Jest config.

## Working with slotted SVGs

### Step one

To slot an SVG into one of our React components, import the `SlottedSVG` component from `@ukic/react` into your React files. This will prevent an error stating that property 'slot' doesn't exist.

```jsx
import { SlottedSVG } from "@ukic/react";
```

### Step two

> Then update your code to replace any instances of `<svg>` with `<SlottedSVG>`.

Reviewer: ```suggestion
Replace any instances of `<svg>` with `<SlottedSVG>`.
```

As an example, when using IcTopNavigation, if your existing code looks like this:

```jsx
<IcTopNavigation appTitle="App title">
  <svg slot="app-icon" path={path} />
</IcTopNavigation>
```

<br />

Then it should be replaced with:

```jsx
<IcTopNavigation appTitle="App title">
  <SlottedSVG slot="app-icon" path={path} />
</IcTopNavigation>

> ```

Reviewer: I think here we can do something like this as it's a long page anyway:

~~As an example, when using IcTopNavigation, if your existing code looks like this:~~

```jsx
// Before
<IcTopNavigation appTitle="App title">
  <svg slot="app-icon" path={path} />
</IcTopNavigation>

// After
<IcTopNavigation appTitle="App title">
  <SlottedSVG slot="app-icon" path={path} />
</IcTopNavigation>
```

The `@ukic/react` package will need to be transformed before you can use these components in Jest tests.

Add a `transformIgnorePatterns` field with the value `["/node_modules/(?!@ukic/react)"]` to your Jest config.

## Working with slotted SVGs

> 

Reviewer 2: ### Step one

> When wanting to slot an SVG into one of the components, you should import SlottedSVG in your React files to prevent an error stating that property 'slot' doesn't exist.

Reviewer 2: ```suggestion
To slot an SVG into one of our React components, import the `SlottedSVG` component from `@ukic/react` into your React files. This will prevent an error stating that property 'slot' doesn't exist.
```

```jsx
import { SlottedSVG } from "@ukic/react";
```

<br />

> Then update your code to replace any instances of `<svg>` with `<SlottedSVG>`.

Reviewer 2: ```suggestion
### Step two
Then, update your code to replace any instances of `<svg>` with `<SlottedSVG>`.
```

As an example, when using IcTopNavigation, if your existing code looks like this:

```jsx
<IcTopNavigation appTitle="App title">
    <svg
        slot="app-icon"
        ...
    >

>         <path ... />

Reviewer 3: I think the path would also be a prop on the svg element here (like you changed for the SlottedSVG), rather than a child\inner content

The `@ukic/react` package will need to be transformed before you can use these components in Jest tests.

Add a `transformIgnorePatterns` field with the value `["/node_modules/(?!@ukic/react)"]` to your Jest config.

## Working with slotted SVGs

> When wanting to slot an SVG into one of the components, you will need to import SlottedSVG in your React files to prevent an error stating that property 'slot' doesn't exist.

Reviewer 3: ```suggestion
When wanting to slot an SVG into one of the components, you should import SlottedSVG in your React files to prevent an error stating that property 'slot' doesn't exist.
```

```jsx
import { SlottedSVG } from "@ukic/react";
```

<br />

Then update your code to replace any instances of `<svg>` with `<SlottedSVG>`.

As an example, when using IcTopNavigation, if your existing code looks like this:

```jsx
<IcTopNavigation appTitle="App title">
    <svg
        slot="app-icon"
        ...
    >
        <path ... />
    </svg>
</IcTopNavigation>
```

<br />

Then it should be replaced with:

```jsx
<IcTopNavigation appTitle="App title">
    <SlottedSVG
        slot="app-icon"
        ...
    >

>         <path ... />

Reviewer 3: should the path by passed as a prop on the SlottedSVG eg something like:
<SlottedSVG slot="app-icon" path={path}></SlottedSvg>
